amazon_payments/views.py

```diff
@@ -540,6 +543,11 @@ def post(self, request, *args, **kwargs):
             order_total = self.get_order_totals(
                 self.request.basket,
                 shipping_method=shipping_method)
+
+            request.basket.calculate_tax(
+                shipping_address
+            )
+            ###
```

Comment:
@sjolicoeur Why this comment? I feel like whitespace is better than `###` in every case I've seen it. Maybe I don't know what you're denoting with the `###` comments?